omod/src/main/compass/sass/patientHeader.scss

```diff
@@ -74,36 +74,6 @@ $green: #51A351;
         top: -15px;
       }
     }
-
-    .colored-message {
-      @include horizontal-list;
-
-      color: white;
-      padding: 2px 4px;
-      border-radius: 1px;
-      margin-top: 4px;
-      font-size: 0.8em;
-    }
-
-    .bordered-colored-message {
-      @extend .colored-message;
-
```

Comment:
OK, having a closer look at it, yes I have changed things here (not only added ```.secondLineFragments {
clear: both;
}```)

The changes are related to the other comment on **omod/src/main/webapp/fragments/patientHeader.gsp** explaining that the position of the `div="identifiers"`, `div class="active-visit-started-at-message"` and `div class="active-visit-message"` wasn't quite right.
Because I have now extracted the new active visit status bar outside the demographics div section, I have had to move the style out of the `.demographics {...}` CSS class as well.